tests/helpers/unittest_base.py

```diff
@@ -139,21 +139,26 @@ def setUp(self, with_data=True, with_consent_codes=False) -> None:
         questionnaire_response_dao._add_codes_if_missing = lambda email: True
         self._consent_questionnaire_id = None
 
-    @staticmethod
-    def setup_storage():
+    def setup_storage(self):
         temp_folder_path = mkdtemp()
+        self.addCleanup(shutil.rmtree, temp_folder_path)
         os.environ['RDR_STORAGE_ROOT'] = temp_folder_path
 
-    @staticmethod
-    def setup_config():
-        if os.environ.get('UNITTEST_CONFIG_FLAG'):
-            return
-        data = read_dev_config(os.path.join(os.path.dirname(__file__), "../../rdr_service/config/base_config.json"),
```

Comment:
```
if os.environ.get('UNITTEST_CONFIG_FLAG'):
            return
```
This will avoid we setup the config for every test case, are you sure you want to remove it?

Reply:
yes so we can keep or overwrite configs

Reply:
Yes we want the configs to be clean for each test.
This does have the added overhead of interacting with the filesystem a lot. If we want to speed this up then we should look into an alternate in-memory `ConfigProvider`